src/test/ui/macros/expr-stmt-nonterminal-tokens.rs

```diff
@@ -0,0 +1,15 @@
+// check-pass
```

Comment:
This file compiles successfully on the latest nightly. I think you need to force recollection of the nonterminals by a proc-macro in order to trigger the panic.

Reply:
Yes, the test assumes existence of the missing tokens panic added in this PR, it won't fail without it.